lms/djangoapps/course_home_api/progress/serializers.py

```diff
@@ -27,6 +27,7 @@ class SubsectionScoresSerializer(ReadOnlySerializer):
     block_key = serializers.SerializerMethodField()
     display_name = serializers.CharField()
     has_graded_assignment = serializers.BooleanField(source='graded')
+    override = serializers.SerializerMethodField()
```

Comment:
Also please add this into the view docstring so we know this field is returned and what it represents.